### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by tommypank** [kubilus1/autoortho#55] [General]
Hey currently having issues completing any of the scenery packs without issues. Europe fails whilst extracting and forces a redownload (even though everything is in the downloads folder.) I have done this a few times now, still the same result.
Another issue I'm having, just tried Aus and this is the error I get once extracted (attached screenshot) 
![Screenshot_20230227_184907](https://user-images.githubusercontent.com/76015940/221655573-b46aa3ad-e198-4dd3-b56a-e74738379a9e.png)

**Comment by kubilus1** [comment]
Interesting.  So you could try that command on the CLI and see if you get any more info.  What OS ver are you using or any other relevant info?  Is there some kind of security or other things to know about the drive you are installed on: such as is this a network mount, etc?

**Reply by tommypank** [reply]
![Screenshot 2023-03-02 204602](https://user-images.githubusercontent.com/76015940/222548012-5159f79c-395a-4c46-8181-d5770dc2b790.png)

**Reply by mikecolive** [reply]
Did you fixed it @tommypank ?

**Reply by tommypank** [reply]
Yeah I managed to fix it, would only work using my local drive not my HDD or SSD. I then used a shortcut to link to my xp12 installation on my SSD. @kubilus1 could this be worth noting in the FAQ that installing to HDD or SSD may cause issues? I also updated python, reinstalled WinFSP, this seemed to sort the .py issues.

In terms of the zip failing, in the new pre release you can prevent the downloads from being purged, delete the problem zip and retry. this will resolve the issue.

**Reply by kubilus1** [reply]
HDD or SSD isn't the issue.  It just needs to be a local NTFS drive.  So not a drive with FAT32 or a remote mount.

**Reply by mikecolive** [reply]
Which version of python? And about winfsp, straight installation without further configurations?

**Comment by tommypank** [comment]
Just reinstalled, downloading as we speak so I will let you know if the issue persists

**Comment by tommypank** [comment]
Local NTFS volumes are required to complete the operation.
ERROR:aoconfig:Traceback (most recent call last):
  File "C:\Users\tompa\AppData\Local\Temp\ON8DBE~1\aoconfig.py", line 261, in scenery_setup
  File "C:\Users\tompa\AppData\Local\Temp\ON8DBE~1\downloader.py", line 354, in extract
  File "C:\Users\tompa\AppData\Local\Temp\ON8DBE~1\subprocess.py", line 369, in check_call
subprocess.CalledProcessError: Command 'mklink /J "E:/SteamLibrary/steamapps/common/X-Plane 12/Custom Scenery\z_aus_pac_00\textures" "E:/SteamLibrary/steamapps/common/X-Plane 12/Custom Scenery\z_autoortho\textures"' returned non-zero exit status 1.

**Reply by kubilus1** [reply]
Yeah I would be shocked if this worked on a network drive under Windows.

**Reply by tommypank** [reply]
If I put a custom scenery folder on C drive and simlink would that be better?

**Reply by kubilus1** [reply]
I have to go out on a limb, since I don't use Windows, but yeah, I think if you created a folder on a local drive, you could point autoortho to that folder.  It may complain if the folder isn't named 'Custom Scenery', but otherwise should work.

You may be able to use shortcuts or symlinks or whatever (not sure on Windows) to reference the created folders to your actual XPlane 'Custom Scenery' folder.   That ought to work.
